Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs

```diff
@@ -61,6 +61,9 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
 		{
 			_buttonLayoutManager?.OnLayout(changed, l, t, r, b);
 			base.OnLayout(changed, l, t, r, b);
+
+			if (!changed && !Control.IsLayoutRequested)
+				Control.ForceLayout();
```

Comment:
Can this be applied to the material one as well? Is there a way to apply this inside the layout manager so that it's generalized?